android/src/main/kotlin/com/mapbox/maps/mapbox_maps/MapboxOptionsController.kt

```diff
@@ -39,6 +42,10 @@ class MapboxOptionsController : _MapboxMapsOptions, _MapboxOptions {
     return ""
   }
 
+  override fun getFlutterAssetPath(flutterAssetUri: String?): String? {
+    return flutterAssetUri?.replace("""^asset://(.+)""".toRegex(), "asset://${flutterAssets.getAssetFilePathBySubpath("$1")}")
```

Comment:
Any chance we would break existing absolute path references? Was it possible for customers to provide full assets path themselves?

Reply:
clients can always provide an absolute path to the asset, however, we'd expect them to not provide them with `asset` scheme in the URL. 
This is done similarly to platform SDKs, is to provide an easy way for customers to refer to a local asset, without having to implement their own logic, or use another 3rd party plugin to get the absolute path.